src/main/java/com/hotels/bdp/circustrain/bigquery/partition/HivePartitionGenerator.java

```diff
@@ -184,13 +186,6 @@ private GeneratePartitionTask(
       return com.google.common.base.Optional.absent();
     }
 
-    private String objectToHiveString(Object partitionValue) {
```

Comment:
I assume this was orphaned code?

Reply:
It was used in `formatter.format(objectToHiveString(partitionValue))`, so if the partition value was blank, then "__HIVE_DEFAULT_PARTITION__" would be returned. But I that wouldn't have helped because it would have been used in BigQuery for querying 
`select * from db.table where partitionKey = "__HIVE_DEFAULT_PARTITION__"`
and that's unlikely to return something.